pkgs/development/python-modules/nvidia-ml-py/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+}:
+
+buildPythonPackage rec {
+  pname = "nvidia-ml-py";
+  version = "11.515.48";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    extension = "tar.gz";
+    hash = "sha256-iNLQu9c8Q3B+FXMObRTtxqE3B/siJIlIlCH6T0rX+sY=";
+  };
+
+  # no tests
+  doCheck = false;
+
+  pythonImportsCheck = [ "pynvml" ];
+
+  patches = [
+    ./0001-locate-libnvidia-ml.so.1-on-NixOS.patch
+  ];
+
+  meta = {
+    description = "Python Bindings for the NVIDIA Management Library";
+    homepage = "https://pypi.org/project/nvidia-ml-py";
+    license = lib.licenses.bsd3;
+    maintainers = with lib.maintainers; [ GaetanLepage ];
```

Comment:
might want to set platforms as this package cannot work on darwin. Should probably be x86_64-linux only.

Reply:
Done :)